### PR TITLE
Added a few reminders and tips in croeseid-testnet.md, mainnet.md and mainnet_validator.md

### PR DESCRIPTION
**Pull request by ysong42**
The reminders and tips are mainly about:

1. Using the specific binary for specific networks.
2. How to generate new validator key.
3. How to fix `too many opened files...` error.

### CODE REVIEW

File: docs/getting-started/croeseid-testnet.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ We officially support macOS, Windows and Linux only. Other platforms may work bu
 The following is the minimal setup for a **validator node**. 
 :::
 
+::: tip Reminder:
+The binary for testnet and the binary for mainnet are two **different** binaries. Please make sure you are using the correct binary.
```
Comment (lezzokafka) [inline docs/getting-started/croeseid-testnet.md:31]:
```suggestion
The binary for _testnet_ and the binary for _mainnet_ are two **different** binaries. Please make sure you are using the correct binary.
```

**Review by lezzokafka — APPROVED** [review]
Looks great, just left a tiny suggestion - Thanks @ysong42  :D

**Comment by ysong42** [comment]
Thanks for the suggestion! It is committed now~ @lezzokafka

**Pull request merged**